kettle-plugins/hadoop-cluster/ui/pom.xml

```diff
@@ -102,6 +102,11 @@
       <version>1.7.1</version>
       <scope>provided</scope>
     </dependency>
+    <dependency>
+      <groupId>commons-configuration</groupId>
+      <artifactId>commons-configuration</artifactId>
+      <version>1.9</version>
```

Comment:
It is good practice to define the version property instead of hardcoding it

Reply:
pushed an updated version. This wasn't in any code we are updating, so I don't even know what impact it will have.